test/unit/lib/cms/aws_credentials_test.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Aws::AwsCredentialsTest < ActiveSupport::TestCase
+  setup do
+    CMS::AwsCredentials.instance.instance_variable_set('@credentials', nil)
+  end
+
+  teardown do
+    CMS::AwsCredentials.instance.instance_variable_set('@credentials', nil)
+  end
```

Comment:
before is in case something already called that earlier. After to leave it clean afterwards.Due to test splitting it will be hard to catch if some other test sets it. And it is hard to debug why credentials were different than expected. So decided to have it both.

Reply:
I see. This would be a good place to put an `around` but it doesn't come out of the box with Minitest 🤷‍♂️